frontend/test/pytest/test_template.py

```diff
@@ -93,7 +174,44 @@ def qaoa_embedding(
     params = [jnp.array([[0.1, -0.3, 1.5], [3.1, 0.2, -2.8]]), jnp.array([1.0, 2.0])]
     interpreted_fn = qml.QNode(qaoa_embedding, device)
     jitted_fn = qjit(interpreted_fn)
-    assert np.allclose(interpreted_fn(*params), jitted_fn(*params))
+
+    interpreted_result = interpreted_fn(*params)
+    jitted_result = jitted_fn(*params)
+
+    assert np.allclose(interpreted_result, jitted_result)
+
+
+@pytest.mark.xfail(reason="Beamsplitter is not supported by lightning.")
```

Comment:
ditto (quantum optics operators include displacements, squeezes, and beamsplitters)